src/providers/export/tensorFlowPascalVOC.test.ts

```diff
@@ -28,6 +32,7 @@ function _base64ToArrayBuffer(base64: string) {
 }
 
 describe("TFPascalVOC Json Export Provider", () => {
+    const testAssets = MockFactory.createTestAssets(10, 1);
     const baseTestProject = MockFactory.createTestProject("Test Project");
     baseTestProject.assets = {
         "asset-1": MockFactory.createTestAsset("1", AssetState.Tagged),
```

Comment:
should we replace these with the `testAssets` created above?

Reply:
No - these testAssets are equivalent to the `NonVisted` assets that are not part of the project file.  The ones defined elsewhere in these tests are those specifically included in the project file as `Visited` or `Tagged`.